src/types/index.ts

```diff
@@ -671,3 +675,12 @@ export interface IDefaultComposableOptions {
 }
 
 export type StatusIconType = typeof ALLOWED_ICON_STATUSES[number];
+
+export type WebSocketChannelName = typeof WEB_SOCKET_CHANNEL_NAMES[number];
```

Comment:
Instead of having to create the array and import it you can just use `ObjectValues<typeof WEB_SOCKET_CHANNELS>`